### PR TITLE
Wrapping up all the header refactoring and updating the affected templat...

### PR DESCRIPTION
**Pull request by weerd**
...es.
Lots of updates! Gonna send this pull request but will NOT merge until Neue is updated. Lots of changes in here that depend on Neue's latest changes.

@aaronschachter @DFurnes @barryclark

### CODE REVIEW

File: lib/themes/dosomething/paraneue_dosomething/templates/campaign_group/node--campaign_group.tpl.php [hunk 1]
```diff
@@ -3,32 +3,38 @@
...
+ * - $title: Title of grouped campaigns page (string).
+ * - $subtitle: Subtitle of grouped campaigns page (string).
+ * - $signup_button: Render array for outputting Signup form button (array).
+ * - $call_to_action: Call To Action of grouped campaigns page (string).
+ * - $scholarship: Scholarship amount (string).
+ * - $partners: Array of partners for grouped campaigns (array).
+ * - $partner_info: Array of information regarding partners for grouped campaigns (array).
  */
+
+// krumo($variables);
```
Comment (DFurnes) [inline lib/themes/dosomething/paraneue_dosomething/templates/campaign_group/node--campaign_group.tpl.php]: KRUMO. :japanese_goblin:

**Comment by DFurnes** [comment]
Looks solid.

**Comment by aaronschachter** [comment]
yea looks ok here too, as far as i can tell it looks like its mostly just a bunch of class changes.

File: lib/themes/dosomething/paraneue_dosomething/templates/campaign_group/node--campaign_group.tpl.php (hunk 1 above)
Comment (weerd) [inline lib/themes/dosomething/paraneue_dosomething/templates/campaign_group/node--campaign_group.tpl.php]: Dammit! I :heart: krumo, but always forget to remove it in places. I'll remove.

**Comment by weerd** [comment]
Cool. going to wait to merge after updates to Neue are approved for release!

**Comment by weerd** [comment]
Ok. Here's goes!

**Pull request merged**